usagov/vote-gov-drupal
# VOTE-2599: Handle NVRF style inheritance issues

## Jira ticket

[VOTE-2599](https://cm-jira.usa.gov/browse/VOTE-2599)

## Description

Most issues were resolved by importing the USWDS styles for each component being used that was not already included in the site. These were the lingering issues being caused by overrides in our theme that are fixed in this PR:
- Font family being overridden by usa-form.scss 
- < p > vertical spacing
- Input field height too small
- typography.scss has a rule that adds spacing above list items which is throwing off the step indicator.
- Usa alerts on confirm page were inheriting extra padding
- Extra spacing above buttons because we are using classes to add margin spacing above them but our theme button overrides is doing the same thing

## Deployment and testing

### QA/Testing instructions

1. lando retune
2. npm run build in theme directory
3. Click through each page of form and confirm all styles are as expected

## Checklist for the Developer




## Checklist for the Peer Reviewers
